opennms-webapp-rest/src/test/java/org/opennms/web/rest/v2/MonitoringLocationRestServiceIT.java

```diff
@@ -73,6 +81,11 @@
 public class MonitoringLocationRestServiceIT extends AbstractSpringJerseyRestTestCase {
     private static final Logger LOG = LoggerFactory.getLogger(MonitoringLocationRestServiceIT.class);
 
+    private static final Integer DEFAULT_LIMIT = 10;
+    public static final Integer DEFAULT_FLAG = null;
+    public static final Integer UNLIMITED = Integer.MAX_VALUE;
+    public static final Integer UNLIMITED_FLAG = 0;
```

Comment:
These flags do not make any sense. Just create locations once, e.g. 12 and just pass the limit to your test method.